synapse/handlers/federation.py

```diff
@@ -2088,9 +2104,14 @@ def _update_auth_events_and_context_for_auth(
                 missing_auth,
             )
             try:
-                remote_auth_chain = yield self.federation_client.get_event_auth(
-                    origin, event.room_id, event.event_id
-                )
+                try:
+                    remote_auth_chain = yield self.federation_client.get_event_auth(
+                        origin, event.room_id, event.event_id
+                    )
+                except RequestSendFailed:
+                    # The other side isn't around or doesn't implement the
+                    # endpoint, so lets just bail out.
+                    return
```

Comment:
conversely, let's not swallow an exception without any logging at all. Let's at least log something.